src/components/SourceFooter.css

```diff
@@ -12,50 +12,39 @@
   user-select: none;
 }
 
-html:not([dir="rtl"]) .source-footer .command-bar {
-  float: right;
+.source-footer__command-bar {
```

Comment:
The goal with this is to reduce the overall complexity in *most* selectors while keeping the different pieces well-scoped. It does mean we type more, but that does give us a clear path with targets and what is doing what.

FTR I did BEM-style since it is just what I know will work in this case. I'm open to any alternative methodologies that net us isolated components with as little leakage as possible.

So why BEM?

* Most targets become a single class or two. Which keeps the overall complexity on selectors down.
* By doing this, we are scoping each component we build to itself. In this case the right sidebar had the same structure class-wise, which led to some of those classes conflicting. Which means tweaking one particular piece could adversely affect the other later if not done very carefully.
* BEM has tooling available to help lint the targets automatically so it is verifiable that alignment with new additions is happening. With some TLC (regex's to exclude) for places where we know we need to go against strict BEM like the `[dir="rtl"]` checks.

It is a big kind of shift in the class naming methodology and is certainly more verbose then what most developers would like. However, it does have a good net benefit on maintainability of the codebase going forward.